1/assignment-week1.py

```diff
@@ -0,0 +1,83 @@
+print ("Week 1 Assignment - Math Equation Evaluator")
+print ("Intern - Shu Wei")
+print ("")
+
+eArray = list(map(str, input("Please enter your input with spaces between them: ").split()))
+
+#Definitions
+aCount = (len(eArray))
+fCount = 3
+nCount = 4
+needle = 0
+
+# Check whether the first two inputs are numbers and the last input is a sign
+try:
+    check1 = int(eArray[0])
+except ValueError:
+    print("Please make sure that the input #1 is a number")
+    exit()
+try:
+    check2 = int(eArray[1])
+except ValueError:
+    print("Please make sure that the input #2 input is a number")
+    exit()
+
+if aCount > 3:
+    CheckNCount = 4
+    while (CheckNCount <= aCount):
+        try:
+            checkN = int(eArray[CheckNCount-1])
+        except ValueError:
+            print ("Please make sure that the input #", CheckNCount, "is a number")
+            exit()
+        CheckNCount = CheckNCount + 2
+else:
+    pass
+
+if aCount >= 3:
+    CheckSCount = 3
+    while (CheckSCount <= aCount):
+        try:
+            checkS = int(eArray[CheckSCount-1])
+        except ValueError:
+            if eArray[CheckSCount-1] == "+" or eArray[CheckSCount-1] == "-" or eArray[CheckSCount-1] == "*" or eArray[CheckSCount-1] == "/":
+                pass
+            else:
+                print("Invalid input #", CheckSCount)
+        else:
+            print ("Please make sure that the input #", CheckSCount, "is a sign")
+            exit ()
+        CheckSCount = CheckSCount + 2
+else:
+    pass
+
+#try:
+#    check3 = int(eArray[aCount-1])
+#except ValueError:
+#    if eArray[aCount-1] == "+" or eArray[aCount-1] == "-" or eArray[aCount-1] == "*" or eArray[aCount-1] == "/":
+#        pass
+#    else:
+#        print("Invalid last sign")
+#else:
+#    print("Please make sure that the last input is a sign")
+#    exit()
+
+# Calculation
+import operator

```

Comment:
usually, in almost all languages, you'd import at the beginning. just a best of practice. Python lets you do it wherever, and there are cases where you'd have it inside some function but usually, imports at the top

Reply:
by the way, I have no idea what import does. I was trying to find a way to convert the text (+,-,*,/) into operations and this was one of the solution.